front-end/src/markup/style.scss

```diff
@@ -0,0 +1,17 @@
+@import './styles/utils/variables';
+@import './styles/utils/mixins';
+@import './styles/utils/reset';
+@import './styles/utils/animations';
```

Comment:
I would recommend you to move these files in utils.scss and just import utils.scss here